util.h

```diff
@@ -59,6 +60,54 @@ extern int UTIL_ensure_dir(const char *dir);
  **/
 extern int UTIL_ensure_path(const char *dir);
 
+/** Support routines for generating telemetry dictionaries, datalogger
```

Comment:
The util grouping is basically for otherwise homeless, but valuable, functionality.  Since the telemetry related functions are 200+ lines it makes sense to give them a formal home.